app/src/main/java/uk/co/philpotter/masteremu/EmuUtils.java

```diff
@@ -0,0 +1,36 @@
+package uk.co.philpotter.masteremu;
```

Comment:
The import statements for Data and Calendar need to be here - the build will fail without them.